tox.ini

```diff
@@ -12,7 +12,7 @@ deps =
 [testenv:coverage]
 usedevelop = true
 basepython =
-    python3.6
+    python3.7
```

Comment:
FWIW, I've been avoiding using Python 3.7 specifically in tox environments. It's still possible that people don't have 3.7 installed yet. I haven't noticed any coverage differences in 3.6 and 3.7.